### PR TITLE
fix(IsAlphaOptions.ignore): convert 'string[]' to 'string'

### PR DESCRIPTION
**Pull request by fedeci**
## Description

This addresses a type bug introduced in #952.

https://github.com/express-validator/express-validator/pull/963#issuecomment-752722295

## To-do list

- [x] I have added tests for what I changed.

- [x] This pull request is ready to merge.

### CODE REVIEW

File: src/options.ts [hunk 1]
```diff
@@ -394,7 +394,7 @@ export interface ContainsOptions {
 }
 
 export interface IsAlphaOptions {
-  ignore?: string[] | RegExp;
+  ignore?: string | RegExp;
 }
```
Comment (fedeci) [inline src/options.ts:398]: This is potentially a breaking change!

Comment (gustavohenke) [reply src/options.ts:398]: Indeed. A way to make it not be breaking would be to test the type of `ignore` in the `isAlpha` implementation, WDYT?

Comment (fedeci) [reply src/options.ts:398]: Yes, I was thinking about that.

File: src/options.ts [hunk 2]
```diff
@@ -394,7 +394,7 @@ export interface ContainsOptions {
...
 
 export interface IsAlphaOptions {
-  ignore?: string | RegExp;
+  ignore?: string[] | RegExp;
```
Comment (gustavohenke) [inline src/options.ts]:
```suggestion
  ignore?: string[] | string | RegExp;
```

should give maximum compatibility 👍

File: src/chain/validators-impl.ts [hunk 3]
```diff
@@ -82,7 +82,9 @@ export class ValidatorsImpl<Chain> implements Validators<Chain> {
     return this.addStandardValidation(validator.isAfter, date);
   }
   isAlpha(locale?: Options.AlphaLocale, options?: Options.IsAlphaOptions) {
-    return this.addStandardValidation(validator.isAlpha, locale, options);
+    const ignore = Array.isArray(options?.ignore) ? options?.ignore.join('') : options?.ignore;
+    delete options?.ignore;
```
Comment (gustavohenke) [inline src/chain/validators-impl.ts:86]: This line can be safely removed, since it'll be overridden below anyway

**Review by gustavohenke — APPROVED** [review]

**Comment by gustavohenke** [comment]
Published as v6.9.1!

**Pull request merged**